Dropdown/Dropdown.module.less

```diff
@@ -54,6 +54,21 @@
 			border-bottom-style: @agate-dropdown-border-style;
 			border-bottom-width: @agate-dropdown-border-width;
 			padding: @agate-dropdown-list-padding;
+
+			.item {
+				border-color: @agate-item-border-color;
+				border-width: @agate-item-border-width;
+
+				.focus({
+					border-color: @agate-item-focus-border-color;
+				});
+
+				&[aria-checked="true"] {
+					&:not(:focus) {
+						color: @agate-dropdown-selected-color;
+					}
```

Comment:
Also, this breaks the UX guide https://zpl.io/29nBwyA  select item keeps the same color

Reply:
Yes, this is true. Thank you for noticing.